app/aptible/torii-adapter.js

```diff
@@ -73,5 +79,15 @@ export default Ember.Object.extend({
       clearSession();
       resolve();
     });
+  },
+
+  identifyToAnalytics(user) {
+    const email = user.get('email');
+    this.get('analytics').identify(email, {
```

Comment:
So in the `query-box` component we use the email b/c we have no other 
identifying information yet. Using email elsewhere was just to make sure 
those people get mapped to real users.

It could be by id, but then I would change the `query-box` component to 
just send a traits object and let it auto-identify